src/main.c

```diff
@@ -19,14 +18,14 @@ void sys_tick_handler(void)
 	if (!collision_detected()) {
 		update_ideal_speed();
 		update_encoder_readings();
-		motor_control();
+		// motor_control();
```

Comment:
This line should not be commented in master (you can comment it locally though). :wink:

Reply:
Ok, :)